src/game/utils/direction.ts

```diff
@@ -0,0 +1,45 @@
+import Position from "../world/position";
+
+export enum Direction {
+  NORTH = "north",
+  SOUTH = "south",
+  EAST = "east",
+  WEST = "west",
+}
+
+export function parseDir(direction: Direction): Position {
+  let dir: Position = new Position(0, 0);
+
+  switch (direction) {
+    case Direction.NORTH:
+      dir.y = 1;
+      break;
+
+    case Direction.SOUTH:
+      dir.y = -1;
+      break;
+
+    case Direction.WEST:
+      dir.x = -1;
+      break;
+
+    case Direction.EAST:
+      dir.x = 1;
+      break;
+
+    default:
+      console.error(`${direction} is not a valid direction.`);
+  }
+
+  return dir;
+}
+
+export function getDirFromVector(x: number, y: number): string {
+  let dir = [];
+  if (y < 0) dir.push("north");
```

Comment:
Probably should be Title Case

Reply:
Also this function name implies that it will return a Direction, not a string

Reply:
Changed function name, but the direction shouldn't be title case here.

Reply:
Why not? If it is for displaying, it should be title case, if not then it should return an array of Directions, not a string

Reply:
Why not? If it is for displaying, it should be title case, if not then it should return an array of `Direction`s, not a string.

Reply:
> When you’re describing a direction, south is lowercase:
The map is behind a secret door on the south wall.
We drove south for 20 miles before we found a gas station.

https://www.quickanddirtytips.com/education/grammar/when-do-you-capitalize-directions